aten/src/ATen/native/mkldnn/xpu/detail/Utils.cpp

```diff
@@ -120,12 +120,12 @@ dnnl::memory::desc get_onednn_md(const at::Tensor& tensor) {
       get_onednn_strides(tensor)};
 }
 
-bool onednn_strides_check(const at::Tensor& src) {
+bool onednn_strides_check(const Tensor& src) {
   auto adims = get_onednn_dims(src);
   int ndims = (int)adims.size();
   auto dims = adims.data();
   auto data_type = static_cast<dnnl_data_type_t>(
-      get_onednn_dtype(src, /*allow_undef*/ true));
+      get_onednn_dtype_include_double(src, /*allow_undef*/ false));
```

Comment:
What is the motivation behind `true->false`?

Reply:
This should be a bug fixing from IPEX by @Stonepia , changing this into false would trigger runtime error if met out-of-expectation error. Otherwise, the error might be hidden.

Reply:
This is because that if we `allow_undef = true`, this data_type might be `undef`, then the consecutive call like below will get wrong result, and cause silent fail.

```C++
// when data_type is undef, the md will be something like `0x1`
dnnl_memory_desc_create_with_strides(&md, ndims, dims, data_type, strides);
```